### PR TITLE
Add trigger and test

### PR DESCRIPTION
**Pull request by iNikitaGricenko**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/surrealdb/Trigger.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "",
+            full = true,
+            code = {
+
+            }
+        )
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/surrealdb/Trigger.java:44]: Please add a proper example

File: src/main/java/io/kestra/plugin/surrealdb/Trigger.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    }
+)
+public class Trigger extends AbstractTrigger implements PollingTriggerInterface, TriggerOutput<Query.Output>, SurrealDBConnectionInterface, QueryInterface {
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/surrealdb/Trigger.java:47]: What's the TriggerOutput interface?

File: src/main/java/io/kestra/plugin/surrealdb/Trigger.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    private boolean useTls = false;
+
+    @Positive
+    @Builder.Default
+    private int port = 8000;
+
+    @NotBlank
+    private String host;
+
+    @NotBlank
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/surrealdb/Trigger.java:59]:
```suggestion
```

File: src/main/java/io/kestra/plugin/surrealdb/Trigger.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    private String username;
+
+    @NotBlank
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/surrealdb/Trigger.java:62]:
```suggestion
```

File: src/test/java/io/kestra/plugin/surrealdb/SurrealDBTest.java
Comment (loicmathieu) [inline src/test/java/io/kestra/plugin/surrealdb/SurrealDBTest.java:1]: Please revert your changes on this class as the default container launched by the docker compose used in the CI has no authentication

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/surrealdb/Trigger.java (hunk 1 above)
Comment (iNikitaGricenko) [reply src/main/java/io/kestra/plugin/surrealdb/Trigger.java:44]: Oh, i totally forgot

**Review by loicmathieu — APPROVED** [review]
LGTM

**Pull request merged**